catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -472,6 +474,20 @@ public Object visit(PropertyIsGreaterThanOrEqualTo filter, Object delegate) {
             return lessThanOrEqual.accept(this, delegate);
         }
 
+        //Special case for handling temporal queries
+        if (literal instanceof String && ((String) literal).matches(
+                "RELATIVE\\(P[0-9]+DT[0-9]+H[0-9]+M\\)")) {
```

Comment:
Oh yeah I guess this would break if we got a request that didn't match exactly how Intrigue was presenting it. I'll change it